libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -1425,6 +1427,23 @@ - (void)finalizeAuthCompletion:(SFSDKOAuthClient *)client
     }
 }
 
+- (void)retrieveUserPhotoIfNeeded:(SFUserAccount *)account{
+    if (account.idData.pictureUrl) {
+        NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:account.idData.pictureUrl];
```

Comment:
Could we use `SFRestRequest` and `SFRestAPI` here instead of `NSMutableURLRequest` and `SFNetwork`?

Reply:
We need `SFRestAPI` to take care of refresh too.